recipes/openmm-torch/meta.yaml

```diff
@@ -0,0 +1,98 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "openmm-torch" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openmm/openmm-torch/archive/v{{ version }}.tar.gz
+  sha256: 6ff3c2d903729eb9145c02e694fe943c52cf9263736f9db26bb84511fbb2d71d
+
+build:
+  number: 0
+  skip: true  # [win or (linux and cuda_compiler_version in (undefined, 'None'))]
```

Comment:
Is this skipping CPU-only Linux builds? Is this intentional or am I misunderstanding it?

Reply:
Intentional, since this plugin is only viable through GPU acceleration.

Reply:
This plugin should work fine without a GPU.  Large models may be very slow without a GPU, but that's another matter.

Reply:
Hm, but since OpenMM does not offer a CPU package, `cudatoolkit` is going to be listed anyway, so there's no benefit (for now) to build CPU versions, right?

Reply:
True.  As long as this doesn't prevent people without CUDA from installing the plugin, that's fine.